x-pack/plugins/ml/server/lib/alerts/alerting_service.ts

```diff
@@ -145,7 +146,10 @@ export function buildExplorerUrl(
       },
     },
   };
-  return `/app/ml/explorer/?_g=${encodeURIComponent(
+
+  const spacePathComponent: string = !spaceId || spaceId === 'default' ? '' : `/s/${spaceId}`;
```

Comment:
a `DEFAULT_SPACE_ID` constant can be found in `'@kbn/spaces-plugin/common'`

Reply:
updated in https://github.com/elastic/kibana/pull/160899/commits/faafc8ef697759991b984e8533b5decc55897db6